implementations/rust/ockam/ockam_api/src/nodes/service/kafka_services.rs

```diff
@@ -181,29 +170,39 @@ impl InMemoryNode {
                 ApiError::core("Unable to get flow control for secure channel listener")
             })?;
 
-        let authority_id = self
+        let authority_identifier = self
             .authority
             .clone()
             .ok_or(ApiError::core("NodeManager has no authority"))?;
 
-        {
-            OutletManagerService::create(
-                context,
-                self.secure_channels.clone(),
-                authority_id.clone(),
-                default_secure_channel_listener_flow_control_id,
-            )
-            .await?;
-        }
+        let outlet_policy_expression = if consumer_route.starts_with(Project::CODE) {
+            let project_identifier = self
+                .cli_state
+                .get_node_project(&self.node_name)
+                .await?
+                .identity
+                .ok_or(ApiError::core("NodeManager has no project"))?;
+            Some(kafka_policy_expression(&project_identifier))
+        } else {
+            None
```

Comment:
In case we are not passing through a project, we can add an explicit policy. This policy should be the default one, but it's subject to change, and it would be more robust to repeat it here: `let check_credential_expression =
                    Expr::Ident(format!("{}.{}", SUBJECT_KEY, ABAC_HAS_CREDENTIAL_KEY))`